portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -67,7 +67,7 @@ $.extend(true, doaj, {
                             }
                             displayTree.push(entry);
                         }
-                        displayTree.sort((a, b) => a.display > b.display);
+                        displayTree.sort((a, b) => a.display > b.display ? 1 : -1);
```

Comment:
such a teeny tiny change, but I've learnt a heck of a lot so I can understand it.